### PR TITLE
Update database-schema-conversion-toolkit.md

### PR DESCRIPTION
**Pull request by abhims14**
Updates for DSCT v0.4

### CODE REVIEW

**Comment by abhims14** [comment]
#sign-off

**Comment by Court72** [comment]
@tdoshin

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by tdoshin** [comment]
#sign-off

**Comment by abhims14** [comment]
#sign-off

File: docs/azure-data-studio/extensions/dsct/database-schema-conversion-toolkit.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ ms.topic: conceptual
...
 # Database Schema Conversion Toolkit (DSCT)
 
-Database Schema Conversion Toolkit (DSCT) is a set of extensions for [Azure Data Studio](../../what-is-azure-data-studio.md) designed to automate database schema migration between different database platforms. This documentation explains the installation of the extensions, common scenarios, and what database platforms are compatible.
+Database Schema Conversion Toolkit (DSCT) is an extension for [Azure Data Studio](../../what-is-azure-data-studio.md) designed to automate database schema conversion between different database platforms. This documentation explains the installation of the extensions, common scenarios, and what database platforms are compatible.
   
 ## Supported sources and targets
   
 - [Oracle to Microsoft SQL](./oracle-to-mssql/database-schema-conversion-toolkit-oracle-to-mssql.md)  
-  Supports migration from Oracle database to Microsoft SQL Server, Azure SQL Database, Azure SQL Managed Instance, and Azure Synapse Analytics.
+  Supports schema conversion from Oracle database to Microsoft SQL Server, Azure SQL Database,and Azure SQL Managed Instance.
```
Comment (Court72) [inline docs/azure-data-studio/extensions/dsct/database-schema-conversion-toolkit.md]:
```suggestion
  Supports schema conversion from Oracle database to Microsoft SQL Server, Azure SQL Database, and Azure SQL Managed Instance.
```

**Pull request merged**